hack/chains/kaniko-cosign-verify.sh

```diff
@@ -35,7 +35,7 @@ title "Cosign verify the image"
 # Save the output data to a file so we can look at it later
 # (Actually we could just pipe it to jq because the text goes to stderr I think..?)
```

Comment:
This seems to work for me:
```
cosign verify ... | yq e . -P -
```

Maybe it's the `show-then-run` command wrapper that doesn't like pipes?

Reply:
Resolving this conversation to avoid noise.